Manager/BackupManager.php

```diff
@@ -84,6 +94,10 @@ public function execute()
             return false;
         }
 
+        if ($successful) {
+            $this->eventDispatcher->dispatch(BackupCompletedEvent::EVENT_NAME, new BackupCompletedEvent());
```

Comment:
Actually if we attach data to the event, we need to do the `$this->processor->cleanUp();` after the event has been sent, here